tests/wasm/closures.js

```diff
@@ -136,3 +136,11 @@ exports.js_store_forgotten_closure = f => {
 exports.js_call_forgotten_closure = () => {
   FORGOTTEN_CLOSURE();
 };
+
+exports.option_works = a => {
+  if (a) {
+    a(2);
+  } else {
+    1;
+  }
+};
```

Comment:
The POSIX specification for files requires that they should have new lines. I think this is breaking the tests.
```suggestion
};

```